pulsar-proxy/pom.xml

```diff
@@ -171,5 +171,12 @@
       <groupId>com.beust</groupId>
       <artifactId>jcommander</artifactId>
     </dependency>
+
+    <dependency>
+      <groupId>${project.groupId}</groupId>
+      <artifactId>bouncy-castle-bcfips</artifactId>
```

Comment:
@zymap Could this be changed into `bouncy-castle-bc` instead of bouncy-castle-bcfips? bc is used more widly.

Reply:
Sure. Updated.